docs/Updating-Software-in-Homebrew.md

```diff
@@ -2,12 +2,15 @@
 
 Did you find something in Homebrew that wasn't the latest version? You can help yourself and others by submitting a pull request to update the formula.
 
-First, check the pull requests in the [homebrew-core](https://github.com/Homebrew/homebrew-core/pulls) repository to make sure there isn't already an open PR. You may also want to look through closed pull requests for the formula, as sometimes formulae run into problems preventing them from being updated and it's better to be aware of any issues before putting significant effort into an update.
+If you want to update software that is either closed source, or a GUI only program you want to follow the guide for Casks. Otherwise follow the guide for Formulae.
```

Comment:
I think this should be moved to below the line that starts "The How To Open a Homebrew Pull Request documentation..." It's not really clear right now what guide you're talking about

Reply:
I put it here to save people from scouring through all possible PRs. Maybe this is better?
```suggestion
If you want to update software that is either closed source, or a GUI only program you want to follow these instructions for Casks. Otherwise follow the instructions for Formulae.
```

Reply:
What about this, instead, as the next paragraph:

> First, check the pull requests in the Homebrew tap repositories to make sure a PR isn't already open. If you're submitting a formula, check [homebrew-core](https://github.com/Homebrew/homebrew-core/pulls). If you're submitting a cask, check [homebrew-cask](https://github.com/Homebrew/homebrew-cask/pulls). You may also want to look through closed pull requests in the repositories, as sometimes packages run into problems preventing them from being updated and it's better to be aware of any issues before putting significant effort into an update.

Reply:
Mine was meant as a quick introduction to the naming, but I think your version would do that too if I just link to the terminology documentation.